### ISSUE TITLE
AppArmor policy for pycache files is ineffective

### ISSUE THREAD

**Issue by eloquence** [freedomofpress/securedrop-client#1235]
Informational finding `TOB-SDW-020` from the [2020 SecureDrop Workstation audit (PDF)](https://media.securedrop.org/media/documents/Trail_of_Bits_SecureDrop_Workstation_Audit_2020.pdf) states that these AppArmor rules for `__pycache__` files: 
 https://github.com/freedomofpress/securedrop-client/blob/c5ca2dc4f65beedbeacce491b97c3ebaf9e19267/files/usr.bin.securedrop-client#L70-L71
are ineffective: "since the whole path is owned by root and due to the used permissions, the user cannot create files in there. On the other side, if the user could create those files, it would allow an attacker who would have arbitrary file write in the `sd-app` VM to create the .pyc file and trigger arbitrary code execution, if the SecureDrop client would be restarted."

The report recommends removing these files or changing the policy to read-only and prebuilding the `__pycache__` files in the template.

**Comment by legoktm** [comment]
Makes sense and as far as these rules are concerned, file ownership (which is correctly root) is much more important than AppArmor.
